1.indicator_analysis.R

```diff
@@ -106,7 +114,7 @@ analyze_first <- function(filename, geography = c("council", "datazone11"),
       group_by(code, year, sex_grp, age_grp) %>% summarise_all(sum, na.rm =T) %>% ungroup()
 
     } else if (measure == "stdrate" & geography == "council") {
-      data_indicator <- data_indicator %>% gather(geolevel, code, ca, hb, scotland) %>% 
+      data_indicator <- data_indicator %>% gather(geolevel, code, ca, hb:scotland) %>% 
```

Comment:
I'm hoping I can copy the trick you used in line 104/112 where the command will be flexible enough to cope with the presence or absence of a column for ADP

Reply:
that looks good